Src/java/cql-to-elm/src/test/resources/org/cqframework/cql/cql2elm/ElmDeserialize/ANCFHIRDummy.json

```diff
@@ -57,6 +57,7 @@
           "dataType" : "{http://hl7.org/fhir}Observation"
         },
         "annotation" : [ {
+          "type" : "Annotation",
```

Comment:
Given there is ELM encoded as JSON out in the wild that is missing this property, whats the implication for backwards compatibility?

Reply:
They will need to fix their JSON. :( 

This would only happen on JXSON-generated JSONs from this project. The JSON generation using JAXB always places a `type` attribute. So, the only affected instances would be from the past 6 months or so. That is why the JSON and JXSON reader/writer could not be used interchangeably until now.  We didn't have enough test cases (with annotations that are not the default type) to see this JXSON bug.